ui/tests/integration/components/pki/config-pki-test.js

```diff
@@ -98,4 +99,38 @@ module('Integration | Component | config pki', function (hooks) {
 
     component.submit();
   });
+
+  test('it calls save with the correct arguments for crl', async function (assert) {
+    assert.expect(3);
+    const section = 'crl';
+    this.set('onRefresh', () => {
+      assert.ok(true, 'refresh called');
+    });
+    this.set(
+      'config',
+      this.mockConfigSave((options) => {
+        assert.equal(options.adapterOptions.method, section, 'method passed to save');
+        assert.deepEqual(options.adapterOptions.fields, ['expiry', 'disable'], 'crl fields passed to save');
+        return resolve();
+      })
+    );
+    this.set('section', section);
+    await render(hbs`<Pki::ConfigPki @section={{section}} @config={{config}} @onRefresh={{onRefresh}} />`);
+    component.submit();
+  });
+
+  test('it initially sets toggle based on the config value', async function (assert) {
```

Comment:
Could we add a test that ensures the `disable` value is what we expect based on crlConfig? Sorry if it exists and I missed it